components/AssignmentsTable.js

```diff
@@ -1,41 +1,28 @@
 import React from 'react'
-import Link from './Link'
-import TableHeaders from './common/TableHeaders'
+import Table from './common/Table'
+
+const tableSchema = {
+  'headers': {
```

Comment:
Since we are declaring the schema on the component, would it make more sense to move the display toolTip in this object as a boolean make the order or properties the implicit column order?

Reply:
We decided to do this in a future refactor